.github/workflows/build.yml

```diff
@@ -4,9 +4,9 @@ on:
   push:
     branches:
       - master
-
+  pull_request:
```

Comment:
```suggestion
  pull_request:
    branches:
      - master
```

Reply:
Okay but isn't it overkill to specify to only build PRs to master?